### PR TITLE
Issue #2: Catch exceptions in Utilities::getEntityDataCount()

### LINKED ISSUE #2: Catch exception from EFQ, show warning
Catch exceptions in Utilities::getEntityDataCount(), and show warning with drush_log().

### PR DESCRIPTION
**Pull request by donquixote**
Fixes #2.

### CODE REVIEW

**Comment by donquixote** [comment]
@vever001 my grumpy is not working atm, so I ignored the checks. You can clean this up if needed.

File: src/Utilities.php [hunk 1]
```diff
@@ -33,7 +35,23 @@ public static function getEntityDataCount($entity_type, $bundle = NULL) {
...
+      // @todo Is this the recommended way to do this?
+      drush_log(
+        dt(
+          "Exception while counting !type entities:\n!message",
+          [
+            '!type' => ($bundle === NULL) ? $entity_type : "$entity_type:$bundle",
+            '!message' => $e->getMessage(),
+          ]),
+        LogLevel::WARNING);
+      return -1;
```
Comment (vever001) [inline src/Utilities.php:49]: Thanks for this @donquixote 
It looks good to me, though I'd personally format it as follows:
```
drush_log(dt('Exception while counting @type entities: !message', [
  '@type' => ($bundle === NULL) ? $entity_type : "$entity_type:$bundle",
  '!message' => $e->getMessage(),
]), LogLevel::WARNING);
```

There is also `drush_set_error()` but maybe we should only throw a warning, not error?
I don't have a strong opinion on this.

Comment (donquixote) [reply src/Utilities.php:49]: I don't have a strong opinion about the formatting.

For the warning vs error, I would say: Warning if operation can proceed afterwards, error if we have to give up and stop.

**Comment by vever001** [comment]
Thanks @donquixote :+1:

**Pull request merged**